### PR TITLE
NIC98

### PR DESCRIPTION
**Pull request by petele**
Changes proposed in this pull request:
- New in Chrome post for NIC98

### CODE REVIEW

**Comment by petele** [comment]
Just a quick note - WCO was cut at the last minute. May need to update this once again.

File: site/en/blog/new-in-chrome-98/index.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
+---
+title: New in Chrome 98
+description: >
+  Chrome 98 is rolling out now! If you’ve signed up
+  for the Auto Dark Theme Origin Trial, there’s a new way to opt out at a per
```
Comment (jpmedley) [inline site/en/blog/new-in-chrome-98/index.md:5]:
```suggestion
  for the Auto Dark Theme origin trial, there’s a new way to opt out at a per
```

File: site/en/blog/new-in-chrome-98/index.md [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+tags:
+  - new-in-chrome
+  - chrome-98
+---
+
+{% YouTube id='TODO' %}
+
+Here's what you need to know:
+
+* If you’ve tried the Auto Dark Theme Origin Trial, there’s a new way to
```
Comment (jpmedley) [inline site/en/blog/new-in-chrome-98/index.md:24]:
```suggestion
* If you’ve tried the Auto Dark Theme origin trial, there’s a new way to
```

File: site/en/blog/new-in-chrome-98/index.md [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+element.
+
+```css
+:root {
+  color-scheme: only light;
+}
+```
+
+But the main advantage of this method, is that you can opt out on a
+per-element basis, by applying a specific style to that element.
```
Comment (jpmedley) [inline site/en/blog/new-in-chrome-98/index.md:58]:
```suggestion
per-element basis by applying a specific style to that element.
```

File: site/en/blog/new-in-chrome-98/index.md [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+
+```css
+.only-light,
+#my-element {
+  color-scheme: only light;
+}
+```
+
+Check out the section on [How to opt-out of Auto Dark Theme][dcc-autodark-optout]
```
Comment (jpmedley) [inline site/en/blog/new-in-chrome-98/index.md:67]: Section? I was looking for it on this page.

File: site/en/blog/new-in-chrome-98/index.md [hunk 5]
```diff
@@ -0,0 +1,148 @@
...
+
+In Chrome 98, there’s now [support for COLRv1][dcc-colrv1], an evolution of
+the COLRv0 font format. It adds gradients, compositing and blending, and
+improved shape reuse for crisp and compact font files that compress well.
+
+A color font makes visual highlights, headlines, and banners really pop out.
+
+{% Img src="image/0g2WvpbGRGdVs0aAPc6ObG7gkud2/qTY5GIw3dMKnQARZZeGx.png", alt="Headline using COLRv1 fonts", width="800", height="661" %}
+
+One of my favorite use cases for COLRv1, means consistent emojis across all
```
Comment (jpmedley) [inline site/en/blog/new-in-chrome-98/index.md:80]:
```suggestion
One of my favorite use cases for COLRv1 means consistent emojis across all
```

File: site/en/blog/new-in-chrome-98/index.md [hunk 6]
```diff
@@ -0,0 +1,148 @@
...
+
+In just a few months, we'll hit Chrome 100, a **three** digit version number.
+Any code that checks version numbers, or parses the UA string, should be
+checked to make sure it handles three digits.
+
+There's a flag called [`#force-major-version-to-100`][cr-100-flag] that
+will change the current version number to 100, so you can make sure
+everything works as expected.
+
+{% Img src="image/0g2WvpbGRGdVs0aAPc6ObG7gkud2/4drS8JxKXgtzSAxyM6WF.png", alt="Chrome flags page highlighting new #force-major-version-to-100 option", width="800", height="533" %}
```
Comment (jpmedley) [inline site/en/blog/new-in-chrome-98/index.md:102]:
```suggestion
{% Img src="image/0g2WvpbGRGdVs0aAPc6ObG7gkud2/4drS8JxKXgtzSAxyM6WF.png", alt="Chrome flags page highlighting the new #force-major-version-to-100 option", width="800", height="533" %}
```

File: site/en/blog/new-in-chrome-98/index.md [hunk 7]
```diff
@@ -0,0 +1,148 @@
...
+
+Of course there's plenty more.
+
+[CORS preflight requests][cs-preflight] are now sent ahead of private
+network requests for subresources, asking for explicit permission from the
+target server.
+
+There’s a new origin trial for [Region Capture][ot-region-capture], an API for
+cropping a self-capture video track. Ideal for screen sharing in video
+conferencing  apps so you don’t create a hall of mirror effect!
```
Comment (jpmedley) [inline site/en/blog/new-in-chrome-98/index.md:114]:
```suggestion
conferencing apps so you don’t create a hall of mirror effect!
```

File: site/en/blog/new-in-chrome-98/index.md [hunk 8]
```diff
@@ -0,0 +1,148 @@
...
+
+Window overlay controls had been planned for Chrome 98, but due to issues found
+at the last minute and did not make it in. We are still working on it, and
```
Comment (jpmedley) [inline site/en/blog/new-in-chrome-98/index.md:117]:
```suggestion
at the last minute, did not make it in. We are still working on it, and
```

**Review by jpmedley — CHANGES_REQUESTED** [review]

File: site/en/blog/new-in-chrome-98/index.md (hunk 4 above)
Comment (petele) [reply site/en/blog/new-in-chrome-98/index.md]: modified this.

**Review by jpmedley — APPROVED** [review]

**Pull request merged**